product_docs/docs/pgd/5/raft/03_migrating_to_raft_subgroups.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Migrating to Raft subgroups
+---
+
+This section assumes that you have an already running cluster and wish to introduce Raft subgroups to your cluster.
+
+
+## Migrating to Raft subgroups (without TPA)
+
+A number of configuration steps need to be taken to enable Raft subgroups to an existing cluster.
+
+* A “top-level” group needs to be defined for all nodes in the PGD cluster. Use bdr.create_node_group
+* A subgroup needs to be created for each location. Use bdr_create_node_group
+* Each location’s subgroup should be added to the “top-level” group. Use bdr.join_node_group(), bdr.switch_node_group() or bdr.alter_node_group().
+* Each node at each location should be added to their location’s subgroup. Use bdr.join_node_group(), bdr.switch_node_group() or bdr.alter_node_group().
+* Each of the location’s subgroups should be altered to enable Raft for the location. Use `bdr.alter_node_group_option()` setting the `enable_raft` option to true.
+
+#### How to enable subgroup Raft node group:
+
+```sql
+SELECT bdr.alter_node_group_option('$group_name', 'enable_raft', 'true');
+```
+
+#### How to disable subgroup Raft for a node group:.
+
+```sql
+SELECT bdr.alter_node_group_option('$group_name', 'enable_raft', 'false');
+```
```

Comment:
Note that this is not currently possible. And my hopes for getting support for disabling Raft for sub-groups into the PGD 5.1 release is vanishing, ATM.